src/bin/nile-rs/cli.rs

```diff
@@ -67,6 +67,9 @@ pub enum Commands {
 
     #[clap(about = "Query the registered accounts from the given network")]
     GetAccounts(GetAccounts),
+
+    #[clap(about = "Remove artifacts from workspace")]
```

Comment:
```suggestion
    #[clap(about = "Remove artifacts and contract deployments from workspace")]
```

I feel users can delete registered accounts by mistake with this, so I would not remove accounts but just contract deployments, or at least I think is worth adding a flag the user needs to manually add for deleting the registered accounts.

Reply:
That's true, I think adding a flag like `--all` would be great to remove the registered contracts as well